### PR TITLE
no query info when measurement tools are opened

### PR DESCRIPTION
**Pull request by rmichaelis**
Please review

### CODE REVIEW

**Comment by rmichaelis** [comment]
No comment about this one ?

File: geoportailv3/static/js/query/querydirective.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ app.queryDirective = function(appQueryTemplateUrl) {
     scope: {
       'map': '=appQueryMap',
       'infoOpen': '=appQueryOpen',
-      'appSelector': '=appQueryAppselector'
+      'appSelector': '=appQueryAppselector',
+      'measureOpen': '=appQueryMeasureopen'
```
Comment (elemoine) [inline geoportailv3/static/js/query/querydirective.js]: To me the query directive should not know about the measure component. Why not use `appQueryActive` as we do in other directives?

**Comment by elemoine** [comment]
I just added a comment suggesting to change the attribute name to avoid coupling the query component to the measure component.

**Comment by rmichaelis** [comment]
Yes you are right. 
This will be also used during the development of the drawing tools.

**Pull request merged**